Makefile

```diff
@@ -101,5 +101,4 @@ install-snap-dependencies: check-snap-package
 install-dependencies: install-snap-dependencies
 ## install-dependencies: Install all the dependencies
 
-
-
+check: juju-unit-test
```

Comment:
@hmlanigan `juju-unit-test` seems to be quite a lightweight test suite.

Shall we put something else here? Or is this good for now? What do you think?

Reply:
Imo best option should be to keep it light in Jenkins, and make more github actions required

Reply:
This set of tests will get bigger over time, but okay for now.